openapi/openapi.yaml

```diff
@@ -604,6 +604,199 @@ paths:
               schema:
                 $ref: "#/components/schemas/AnalysisResponseError"
 
+  /hardware-envrionment:
```

Comment:
`typo` in environment. And shouldn't be plural?

Reply:
This depends how we design API. To be consistent with the current endpoints, like `/analyze`, this can stay as is.

My personal preference is structuring endpoints in a way they narrow down to a concrete item requested (hence `/analyze` is in singular):

* `/analyze`
* `/analyze/{analysis_id}`
* `/analyze/{analysis_id}/log`
* `/analyze/{analysis_id}/status`
* ...

I'm open to redesigning this - let's make this part of https://trello.com/c/AE8sby1A/797-design-rest-api-on-user-api-so-that-users-can-easily-integrate